subprojects/internal-testing/src/main/resources/valid-precondition-combinations.csv

```diff
@@ -88,6 +91,10 @@ IntegTestPreconditions$Java8HomeAvailable,IntegTestPreconditions$Java17HomeAvail
 IntegTestPreconditions$JavaHomeWithDifferentVersionAvailable
 IntegTestPreconditions$MoreThanOneJava8HomeAvailable
 IntegTestPreconditions$LowestSupportedLTSJavaHomeAvailable,IntegTestPreconditions$HighestSupportedLTSJavaHomeAvailable
+IntegTestPreconditions$MoreThanOneJavaHomeAvailable
+IntegTestPreconditions$MoreThanOneJava8HomeAvailable,IntegTestPreconditions$Java9HomeAvailable
+IntegTestPreconditions$IsNoDaemonExecutor
```

Comment:
What is the reason to include the single conditions? Do you want to be able to forbid the singular application of a condition? Otherwise, I'd change it so, that singular sets are inherently valid. And remove all single entries here.